app/app.go

```diff
@@ -25,6 +27,8 @@ type App struct {
 	Hubs                        []*Hub
 	HubsStopCheckingForDeadlock chan bool
 
+	Jobs *jobs.JobServer
```

Comment:
minor nit, why is this plural?  isn't it a single job server?

Reply:
Single server, multiple jobs. It just seemed more natural to go from `jobs.DoThing` to `a.Jobs.DoThing`.